matminer/featurizers/site.py

```diff
@@ -464,14 +472,31 @@ def featurize(self, struct, idx):
 
         cn_fingerprint = []
 
+        if self.chem_info is not None:
+            absdeltas = {}
+            for prop in self.chem_props:
+                absdeltas[prop] = 0
+            Nwt = 0
+            elem_cent = struct.sites[idx].species_string
```

Comment:
suggest rename ``elem_cent`` to ``elem_central``

Reply:
Note also that the species string can be problematic

Often the species is something like ``Al3+`` but your property dict might have values for elements only (e.g., a value for ``Al``). Ideally if you don't have a value for the specific species, the code would fall back on just using the element specific value if you have it. e.g., if you didn't define a value for ``Al3+`` but did define a value for ``Al``, it will just use the ``Al`` value.